notes/v1.0.0-beta1.md

```diff
@@ -0,0 +1,211 @@
+* `ecctl deployment upgrade`       Updates a deployment from a file definition, allowing certain flag overrides
+
+### Added a plan cancel for the deployment resources
+
+* `ecctl deployment plan cancel`
+
+### Added all deployment resource commands
+
+Added all the deployment resource actions which are available in the API:
+
+* `ecctl deployment resource delete`            Deletes a previously shut down deployment resource
+* `ecctl deployment resource restore`           Restores a previously shut down deployment resource
+* `ecctl deployment resource shutdown`          Shuts down a deployment resource by its type and ref-id
+* `ecctl deployment resource start`             Starts a previously stopped deployment resource
+* `ecctl deployment resource start-maintenance` Starts maintenance mode on a deployment resource
+* `ecctl deployment resource stop`              Stops a deployment resource
+* `ecctl deployment resource stop-maintenance`  Stops maintenance mode on a deployment resource
+* `ecctl deployment resource upgrade`           Upgrades a deployment resource
+
+### Added elasticsearch keystore commands
+
+Manages an Elasticsearch cluster's keystore
+
+* `set`         Updates an Elasticsearch cluster keystore with the contents of a file
+* `show`        Shows an Elasticsearch cluster's keystore settings
+
+## Bug fixes
+<!-- Add the list of bug fixes, remove otherwise.-->
+
+### init command now skips tls validation
+
+It was reported that in some instances `ecctl init` would return with an error even after providing valid
+credentials, this was mainly due to ecctl performing TLS validation where a lot of ECE installations have
+self signed certificates. It now will skip TLS Verification by default.
+
+### init command now writes the JSON config without returning an error
+
+The default directory for configuration (`~/.ecctl`) was oftentimes not created in the machine where the
+command was being run, which caused the configuration to not be written to that folder. This has been fixed.
+
+### --timeout flag is now honored
+
+Using the `--timeout` global flag now has the desired effect in any of the commands, where before a static 30s
+timeout was used from the default transport even when `http.Client.Timeout` being overriden.
+
+### Unexpected errors are now unpacked instead of returning "unknown error (status xxx)"
+
+Leverages the new unwrapping capabilites of `api.UnwapError` which now unwraps the API errors which previously
+were returned as `unknown error (status xxx)`.
+
+### Fix broken "user key show" command
+
+The command was broken since the UserID wasn't specified and the Key ID was being set instead of the User ID.
+The command now works as expected.
+
+## Docs
+<!-- Add the list of doc changes, remove otherwise.-->
+
+### Added ASCIIDoc for commands
+
+### Added auto-completion instructions to the docs
+
+### Detailed changes
+<!-- AUTOGENERATED COMMIT LIST -->
+
+97c3985 Adding ecctl icon to repo (#111)
```

Comment:
Same comment about the newlines. Unless this isn't as important, in which case leave as is :)